akka-stream/src/main/scala/akka/stream/impl/io/TcpStages.scala

```diff
@@ -385,7 +385,7 @@ private[stream] object ConnectionSourceStage {
         cause match {
           case _: SubscriptionWithCancelException.NonFailureCancellation =>
             log.debug(
-              "Not aborting connection from {}:{} because downstream cancelled stream without failure",
+              "Closing connection from {}:{} because downstream cancelled stream without failure",
```

Comment:
Should we perhaps remove "without failure" as well, it is just a normal downstream cancel, right?

Reply:
might be helpful to keep it since a cancellation caused by a failure would abort the connection rather than closing it